### DISCUSSION TITLE
附加功能-下载视频中能否使用MP4格式进行下载

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#1674] [功能建议 / Ideas]
有一次下载视频当素材（选的flv格式，1080p，用链接下载的），打开以后发现，视频在复杂部分时会比原视频模糊（轻度马赛克，还是挺影响观感的），不知道是不是因为格式问题（不专业的眼神.jpg）
希望可以新增一个MP4格式awa
浏览器：runningcheese魔改版FirefoxV85.0
插件版本：V1.12.0
视频：[https://www.bilibili.com/video/BV1oU4y1H7Pw](url)

**Comment by Geeyun-JY3** [comment]
看常见问题：#1301
